legend-engine-xt-persistence-grammar/src/main/java/org/finos/legend/engine/language/pure/dsl/persistence/compiler/toPureGraph/PersistenceCompilerExtension.java

```diff
@@ -101,14 +114,65 @@ public List<Function2<Trigger, CompileContext, Root_meta_pure_persistence_metamo
             else if (trigger instanceof CronTrigger)
             {
                 CronTrigger cronTrigger = (CronTrigger) trigger;
-                return new Root_meta_pure_persistence_metamodel_trigger_CronTrigger_Impl("", null, compileContext.pureModel.getClass("meta::pure::persistence::metamodel::trigger::CronTrigger"))
-                        ._minutes(cronTrigger.minutes)
-                        ._hours(cronTrigger.hours)
-                        ._dayOfMonth(cronTrigger.dayOfMonth)
-                        ._month(cronTrigger.month)
-                        ._dayOfWeek(cronTrigger.dayOfWeek);
+                return new Root_meta_pure_persistence_metamodel_trigger_CronTrigger_Impl("", null, compileContext.pureModel.getClass("meta::pure::persistence::metamodel::trigger::CronTrigger"))._minutes(cronTrigger.minutes)._hours(cronTrigger.hours)._dayOfMonth(cronTrigger.dayOfMonth)._month(cronTrigger.month)._dayOfWeek(cronTrigger.dayOfWeek);
```

Comment:
Was the reformatting intentional ? The previous multiline formatting was easier on the eye !

Reply:
Agreed -- restored!